src/warnet/scenarios/tx_flood.py

```diff
@@ -8,7 +8,7 @@
 
 
 def cli_help():
-    return "Make a big transaction mess"
+    return "Make a big transaction mess. Options: [--interval ]"
```

Comment:
looks like this is a manual process? would be nice if we could automatically pipe through all of the commands, otherwise I suspect we will keep running into this.

Reply:
See #429

Reply:
Seems better to drop this change from this PR and merge #429 ?  I left a quick review on 429 but my comments are non-blocking

Reply:
oops i did just hard code the option on main, couldnt get #429  to work